changelog/replacementDchar.dd

```diff
@@ -0,0 +1,41 @@
+# Change autodecode from throwing `UTFException` on invalid encodings to returning `std.utf.replacementDchar` `('\uFFFD')` instead.
+
+When `char` and `wchar` slices are iterated, they are autodecoded meaning the UTF-8 and
+UTF-16 code unit sequences are automatically consumed and decoded into `dchar` code points.
+If an invalid sequence is encountered, a
+$(LINK2 https://dlang.org/phobos/std_utf.html#UTFExceptions,`UTFException`)
+is raised.
+
+This change instead causes
+$(LINK2 https://dlang.org/phobos/std_utf.html#replacementDchar, `std.utf.replacementDchar`)
+to be the decoded result of an invalid sequence.
+
+If the old behavior is desired, the autodecoding iteration can be replaced with
+$(LINK2 https://dlang.org/phobos/std_utf.html#byDchar, `std.utf.byDchar()`)
+to decode into `dchar`s followed by
+$(LINK2 https://dlang.org/phobos/std_algorithm_iteration.html#.filter, `std.algorithm.filter()`)
+to detect the `replacementDchar` and throw the exception, as in the following example:
```

Comment:
Please add some text here how this approach will not work with any valid UTF streams that already contain the replacement character, and they will be indistinguishable from invalid streams.

Reply:
done